src/components/surveys/survey-design/question-edit/QuestionEditPhone.tsx

```diff
@@ -213,10 +213,10 @@ const QuestionEditPhone: FunctionComponent<QuestionEditProps> = ({
       return <SelectPhoneBottomMenu step={step as ChoiceQuestion} onChange={s => onChange(s)} />
     }
     if (step.type === 'overview') {
-      return <StyledStartButton>Start</StyledStartButton>
+      return <></>
     }
     if (step.type === 'completion') {
-      return <StyledStartButton>Exit</StyledStartButton>
+      return <></>
     }
```

Comment:
It doesn't look like `StyledStartButton` is used anywhere else, so should the styled component be removed as well?

Reply:
Cleaned up and removed unused components. Thanks!